contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -536,6 +536,14 @@ class IndexedDBResource {
       return this.table.delete(id);
     });
   }
+
+  first() {
+    return this.table.toCollection().first();
+  }
+
+  indexNotEqual(index, value) {
```

Comment:
I added this function because it seems that it's not possible to access `notEqual` directly on IndexedDBResource after calling `where` on it. I am thinking whether it would make sense to somehow expose Dexie methods so we can use its API instead of adding more and more custom methods on our wrapper like here. Or is it possible to use it somehow now?

Reply:
Can always just call `SessionResource.table.where(index).notEqual(value)` from wherever you are invoking it.

Reply:
Thank you, accessing the table directly in this way makes sense. I am also going to remove the `first` method from `IndexedDBResource`.

Reply:
We'll just need to be careful to know the difference between using `Resource.table.where` and our own implementation for `Resource.where`